### PR TITLE
New Grouped AVP + Changes in current commands

### PR DESCRIPTION
**Pull request by bilotto**
I have included some new grouped AVP and associated with the corresponding messages. These are all tested and working fine.

### CODE REVIEW

File: src/diameter/message/commands/aa.py [hunk 1]
```diff
@@ -372,6 +381,9 @@ def __post_init__(self):
         setattr(self, "redirect_host", [])
         setattr(self, "proxy_info", [])
         setattr(self, "route_record", [])
+        setattr(self, "media_component_description", MediaComponentDescription())
+        setattr(self, "supported_features", SupportedFeatures())
```
Comment (mensonen) [inline src/diameter/message/commands/aa.py:385]: Should not be initialised manually.

File: src/diameter/message/commands/credit_control.py [hunk 2]
```diff
@@ -314,6 +361,10 @@ def __post_init__(self):
         setattr(self, "service_parameter_info", [])
         setattr(self, "proxy_info", [])
         setattr(self, "route_record", [])
+        setattr(self, "supported_features", SupportedFeatures())
+        setattr(self, "qos_information", QosInformation())
+        setattr(self, "default_eps_bearer_qos", DefaultEpsBearerQos())
```
Comment (mensonen) [inline src/diameter/message/commands/credit_control.py:366]: Should not be initialised manually.

**Review by mensonen — CHANGES_REQUESTED** [review]
You're changing a design pattern here, by always setting certain attributes, e.g.:

```python
        setattr(self, "media_component_description", MediaComponentDescription())
```

Doing this will result in Media-Component-Description AVP *always* being present in every AA message, even if not actually used. The reason why some attributes are initialised in the `__post_init__` is that they need to start as empty lists. It's otherwise not needed at all.

Already having 
```python
        AvpGenDef("media_component_description", AVP_TGPP_MEDIA_COMPONENT_DESCRIPTION, VENDOR_TGPP, type_class=MediaComponentDescription)
```

is sufficient to ensure that these grouped AVPs are initialised correctly, if and when needed.

If you can remove the initialisation of media_component_description and supported_features, as well as the similar pattern in Credit-Control, then I can merge this.

**Comment by bilotto** [comment]
@mensonen Understood! Thank you for the feedback. Please review the updates.

**Pull request merged**